protocol/chat.go

```diff
@@ -40,6 +41,28 @@ const (
 	FirstMessageTimestampNoMessage = 1
 )
 
+const (
+	MuteFor1MinDuration   = 60
+	MuteFor15MinsDuration = 900
+	MuteFor1HrsDuration   = 3600
+	MuteFor8HrsDuration   = 28800
+	MuteFor1WeekDuration  = 604800
+)
+
+const (
+	MuteFor15MinsType requests.MutingType = iota + 1
+	MuteFor1HrsType
+	MuteFor8HrsType
+	MuteFor1WeekType
+	MuteTillUnmuted
+	MuteTill1MinType
+)
+
+type MuteChat struct {
```

Comment:
Are you still using this one? or it can be removed?

Reply:
Sorry forgot to remove it .. Removed